content/en/tracing/setup_overview/compatibility_requirements/php.md

```diff
@@ -44,31 +44,37 @@ PHP APM supports the following SAPI's:
 
 #### Web Framework Compatibility
 
-By default, Datadog **supports all PHP web frameworks** out of the box, which allows you to see traces for spans of supported libraries—for example: database and HTTP clients.
+By default, Datadog **supports all PHP web frameworks** out of the box, with either Framework-Level instrumentation, or Generic Web Tracing.
+
+Framework-Level instrumentation includes tracing of internals methods and framework specific tagging.
+
+Generic web tracing will include a `web.request` span to track latency and errors that originated from the call, in addition to spans for supported libraries — for example: database and HTTP clients.
 
 The following table enumerates some of the frameworks and versions Datadog successfully traces.
 
-**Web frameworks**:
-
-| Module         | Versions      | Support Type               |
-|:---------------|:--------------|:---------------------------|
-| CakePHP        | 2.x           | All supported PHP versions |
-| CodeIgniter    | 2.x, 3.x      | PHP 7+                     |
-| Laravel        | 4.2, 5.x, 6.x | All supported PHP versions |
-| Laravel 8      | 8.x (as of `0.52.0`) | All supported PHP versions |
-| Lumen          | 5.2+          | All supported PHP versions |
-| Symfony 3      | 3.3, 3.4      | All supported PHP versions |
-| Symfony 4      | 4.x           | All supported PHP versions |
-| Symfony 5      | 5.x (as of `0.50.0`) | All supported PHP versions |
-| WordPress      | 4.x, 5.x      | PHP 7+                     |
-| Zend Framework | 1.12          | All supported PHP versions |
-| Yii            | 1.1, 2.0      | All supported PHP versions |
-| Drupal         |               | All supported PHP versions |
-| Magento        | 1, 2          | All supported PHP versions |
-| Phalcon        | 1.3, 3.4      | All supported PHP versions |
-| Slim           | 2.x, 3.x, 4.x | All supported PHP versions |
-| Neos Flow      | 1.1           | All supported PHP versions |
-| FuelPHP        | 1.1           | PHP 7+                     |
+**Web frameworks with internals tracing**:
```

Comment:
I think we're being clear enough elsewhere in the doc we don't need to edit this line
```suggestion
**Web frameworks**:
```